app/controllers/deploys_controller.rb

```diff
@@ -109,6 +110,11 @@ def destroy
 
   protected
 
+  def deploy_permitted_params
+    @deploy_permitted_params ||=
```

Comment:
no need for the caching in `@deploy_permitted_params` ... it is not reused ... or is it ?